### DISCUSSION TITLE
세그멘테이션 오류 | Jae’s Blog

### DISCUSSION THREAD

**Discussion by giscus** [jaenam615/jaenam615.github.io#12] [Announcements]
# 세그멘테이션 오류 | Jae’s Blog

Hydejack is a boutique Jekyll theme for hackers, nerds, and academics, with a focus on personal sites that are meant to impress.

https://jaenam615.github.io/cs/os/2024-03-11-segmentation_fault/

**Comment by moonmisae118** [comment]
게시글 좀많이 올려주세요! 퍼가요~
=======================
